biome.json

```diff
@@ -26,6 +26,9 @@
 			},
 			"correctness": {
 				"useExhaustiveDependencies": "off"
+			},
+			"complexity": {
+				"noForEach": "off"
```

Comment:
Biome recommends using `for...of` instead of `.forEach` for performance and readability reasons.
https://biomejs.dev/linter/rules/no-for-each/

That's nice but this lint rule has a bunch of false positives because of firebase's `QuerySnapshot`, which cannot be refactored to use `for...of`.

To avoid having to ignore every usage of `forEach`, I just disabled this.

https://github.com/KonferCA/HawkHacks-Dashboard/blob/6bd52ffc5926edb3442d2d27428676438631112a/functions/src/teams.ts#L118-L133